ml-proto/winmake.bat

```diff
@@ -0,0 +1,68 @@
+rem Auto-generated from Makefile!
```

Comment:
If this is auto-generated but we're checking it in, we should make sure to clearly documented that people should rerun make and check in the .bat file every time